db/seeds/07_solutions.js

```diff
@@ -1,6 +1,76 @@
 const fakeSolutions = [];
 
-fakeSolutions.push({ name: 'More desks', organization_id: 1 });
+fakeSolutions.push({
+    name: 'More desks',
+    organization_id: 1
+}, {
+    name: 'Projector Repair',
+    organization_id: 2,
+    date: "04_02_19",
+    time: "5:16 PM"
+}, {
+    name: 'New books',
+    organization_id: 10,
+    date: "04_03_19",
+    time: "5:16 PM"
+}, {
+    name: 'Smoke alarms',
+    organization_id: 1,
+    date: "04_04_19",
+    time: "5:16 PM"
+}, {
+    name: 'Science Lab Materials',
+    organization_id: 2,
+    date: "04_05_19",
+    time: "5:16 PM"
+}, {
+    name: 'Art Easels',
+    organization_id: 10,
+    date: "04_08_19",
+    time: "5:16 PM"
+}, {
+    name: 'Desk chairs',
+    organization_id: 1,
+    date: "04_09_19",
+    time: "5:16 PM"
+}, {
+    name: 'Whiteboards',
+    organization_id: 2,
+    date: "04_10_19",
+    time: "5:16 PM"
+}, {
+    name: 'Chalkboard Replacement',
+    organization_id: 10,
```

Comment:
These organization ids should all be 1 or 2 since the DB is only seeded with 2 organizations (and these have to be existing)!